locales/container/en.yml

```diff
@@ -16,6 +17,7 @@ en:
     - summary
     - quit
     timezone: Set Timezone
+    dbbackup: Create Database Backup
```

Comment:
This shouldn't be in the container file. We have other solutions for backup and restore there.

Reply:
This is one of those situations where I wasn't sure, but saw when messing with the `I18n` stuff that there were two versions of this.  So I figured I would add it in both and you could yell at me if I got it wrong (as you did), instead of me forgetting about not adding it and then needing to add it in a follow up/bug fix.

tl; dr  Thanks!  Will remove.